packages/vaults/src/transaction/member/member_create.ts

```diff
@@ -9,11 +9,27 @@ import {TransactionMapperAbstract} from "../transaction_mapper";
 import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
-
+/**
+ * Interface for a transaction that creates a new member.
+ * The memberId is the principal of the user with the default subaccount.
```

Comment:
Do we have a more specifc `type` for this string? What's the string representation of a principal with the default subaccount?

Reply:
Nope. It's an address which is hex string

Reply:
Is there a standard way to get this address? Should we describe that here?

Reply:
added a link to the class from dfinity-agent-js library